tests/code-splitting/subtests/between-new-worker-type/webpack/webpack.config.js

```diff
@@ -11,16 +11,21 @@
  * limitations under the License.
  */
 module.exports = {
-  resolveLoader: {
-    alias: {
-      omt: 'worker-plugin/loader?esModule',
-    },
+  module: {
+    rules: [
+      {
+        test: /\.m?js$/,
+        parser: {
+          worker: [
```

Comment:
@sokra I'm struggling to find docs for this. I got as far as https://webpack.js.org/configuration/module/#ruleparser, but couldn't find out what the values here actually do.

Reply:
IIRC these are a list of "free globals" to consider as worker factories, and `"..."` is the built-in list (`["Worker","SharedWorker","navigator.serviceWorker.register"]`). The possible values would be a good blurb to add to the docs for sure.

Reply:
```suggestion
          // any invocations of these global functions are interpreted as bundle entries:
          worker: [
```